lib/util/Constants.dart

```diff
@@ -0,0 +1,5 @@
+// urls
+const url_udStreamRegister = 'https://www.euclideon.com/contact-us/';
```

Comment:
This is probably beyond the scope of your MVP, but udSDK does allow you to register an account using `udUserUtil_Register`.